chromium_src/chrome/browser/profiles/pref_service_builder_utils.cc

```diff
@@ -10,6 +10,7 @@
 #include "brave/browser/profiles/profile_util.h"
 #include "brave/browser/tor/buildflags.h"
 #include "brave/common/pref_names.h"
+#include "brave/components/brave_ads/browser/ads_service.h"
```

Comment:
is there already a dep for this?

Reply:
There is already a dep. I've now moved the dep inside an `brave_ads_enabled` guard